ouroboros-consensus-byron/test/Test/ThreadNet/RealPBFT.hs

```diff
@@ -620,11 +621,14 @@ prop_setup_coreNodeId numCoreNodes coreNodeId =
 
     params :: PBftParams
     params = realPBftParams dummyK numCoreNodes
-    dummyK = SecurityParam 10   -- not really used
+
+    -- not really used
+    dummyK       = SecurityParam 10
+    dummySlotLen = slotLengthFromSec 20
```

Comment:
Can you explain this to me? I suppose they will be overridden by some other `k` and `SlotLength`, but where/how?

Reply:
Not a clue. I think this test is just a sanity check on some config. You'd have to ask @nfrisby . But I think these values just don't matter.

Reply:
In the scope of these definitions (10 lines up) you see that the property doesn't depend on these. But it does depend on the `ProtocolInfo`, which needs the params, which needs these values. If fields were sufficiently lazy (or similar), we could use `error` here.

Reply:
:+1: